agglayer/grpc/agglayer_grpc_client.go

```diff
@@ -72,6 +77,13 @@ func (a *AgglayerGRPCClient) SendCertificate(ctx context.Context,
 	ctx, cancel := context.WithTimeout(ctx, a.cfg.RequestTimeout.Duration)
 	defer cancel()
 
+	if validatorSignature != nil {
```

Comment:
Maybe it slightly more idiomatic, because it is a slice?

```suggestion
	if len(validatorSignature) > 0 {
```

Reply:
Fix: https://github.com/agglayer/aggkit/pull/738/commits/4de1772703bb2b9a6795ef3ab4e61c4f22f8dd1d